docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
 
-   Grafana generates a link of the snapshot.
+### Update access to an external dashboard link
 
-1. Copy the snapshot link, and share it either within your organization or publicly on the web.
+You can update the access to externally shared dashboard links by following these steps:
 
-If you created a snapshot by mistake, click **Delete snapshot** in the dialog box to remove the snapshot from your Grafana instance.
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. Do one of the following:
+   - Click **Pause access** so that people can't access the dashboard, but the link is maintained.
+   - Click **Resume access** so that people can access the dashboard again.
+   - Click **Revoke access** so that people can't access the dashboard unless a new external link is generated. Confirm that you want to revoke the link.
+1. Click the **X** at the top-right corner to close the share drawer.
 
-#### Delete a snapshot
+### Schedule a report
 
-To delete existing snapshots, follow these steps:
+To share your dashboard as a report, follow these steps:
 
 1. Click **Dashboards** in the main menu.
-1. Click **Snapshots** to go to the snapshots management page.
-1. Click the red **x** next to the snapshot URL that you want to delete.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Schedule a report**.
+1. [Configure the report](ref:configure-report).
+1. Depending on your schedule settings, click **Schedule send** or **Send now**.
 
-The snapshot is immediately deleted. You may need to clear your browser cache or use a private or incognito browser to confirm this.
+You can also save the report as a draft.
 
-### Export a dashboard as JSON
+To manage your reports, navigate to **Dashboards > Reporting > Reports**.
 
-The dashboard export action creates a Grafana JSON file that contains everything you need, including layout, variables, styles, data sources, queries, and so on, so that you can later import the dashboard.
+### Share a snapshot
 
-1. Click **Dashboards** in the main menu.
-1. Open the dashboard you want to export.
-1. Click **Share** in the top-right corner.
-1. Click **Export**.
+A dashboard snapshot publicly shares a dashboard while removing sensitive data such as queries and panel links, leaving only visible metrics and series names. Anyone with the link can access the snapshot.
+
+<!-- is the following para still true? -->
+
+You can publish snapshots to your local instance or to [snapshots.raintank.io](http://snapshots.raintank.io). The latter is a free service provided by Grafana Labs that enables you to publish dashboard snapshots to an external Grafana instance. Anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
```

Comment:
The snapshots.raintank.io option is disabled by default in Cloud. Maybe we should mention the config that enable/disable it here.

Reply:
Happy to add it in or link to it. But what are the steps? I'm not seeing an obvious feature toggle.

Reply:
It's inside the [configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#external_enabled) (not a feature toggle). I'm not saying we should remove this, but if a Cloud user reads this, they won't find these options since they are disabled in the Cloud. I'm not sure how to properly communicate this.

Reply:
Updated!